server/routes/parcels.js

```diff
@@ -8,6 +8,9 @@ router.get('/', ParcelController.getAllParcels);
 router.get('/:id', ParcelController.getParcelById);
 router.put('/:id/cancel', ParcelController.cancelParcel);
 router.put('/:id/', ParcelController.updateParcel);
+router.put('/:id/destination', ParcelController.updateParcel);

```

Comment:
why do you have multiple routes using the same controller?

Reply:
Thanks. I have not implemented it yet and I will create their controllers. I just copied and pasted my earlier  PUT route  since they all update a field in the database table but I will write their separate controllers when I am done with users controllers. Thanks for your feedback.